### ISSUE TITLE
Updating Password Not Working

### ISSUE THREAD

**Issue by BhimAle** [scottyab/secure-preferences#25]
While using one static key , it is working as charm, but my situation is bit different,
need to change the key often.
So I tried to use

        SecurePreferences securePrefs = new SecurePreferences(context, "userpassword", "my_user_prefs.xml");
        securePrefs.handlePasswordChange("newPassword", context);

But not working.

**Comment by scottyab** [comment]
Ok thanks @BhimAle  More correctly here's the example / test that fails.... 

```java
 public void testChangeUserPassword() {
        SecurePreferences securePrefs = new SecurePreferences(getContext(), "myfirstpassword", USER_PREFS_WITH_PASSWORD);
        Editor editor = securePrefs.edit();
        final String key = "pwchgfoo";
        final String value = "pwchgbar";
        editor.putString(key,value);
        editor.commit();
        securePrefs.handlePasswordChange("newPassword", getContext());

        String valueFromNewPassword = securePrefs.getString(key, null);
          assertEquals(value, valueFromNewPassword);
    }
```

**Comment by scottyab** [comment]
@BhimAle this is fixed in master branch and will be included in 0.1.3 release.

**Comment by BhimAle** [comment]
@scottyab ok Thank You very much :)
